segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/AppendProcessor.java

```diff
@@ -252,6 +260,29 @@ public void append(Append append) {
                 });
     }
 
+    @Override
+    public void createTransientSegment(CreateTransientSegment createTransientSegment) {
+        String operation = "createTransientSegment";
+        long traceId = LoggerHelpers.traceEnter(log, operation, createTransientSegment);
+        Collection<AttributeUpdate> attributes = Arrays.asList(
+                new AttributeUpdate(CREATION_TIME, AttributeUpdateType.None, System.currentTimeMillis()),
+                new AttributeUpdate(ATTRIBUTE_SEGMENT_TYPE, AttributeUpdateType.None, SegmentType.TRANSIENT_SEGMENT.getValue())
+        );
+        String transientSegmentName = NameUtils.getTransientNameFromId(createTransientSegment.getParentSegment(), UUID.randomUUID());
```

Comment:
Let's pass in the `Writer Id` here and then use that and a random UUID (inside the NameUtils) to generate the name. This will help us in the future to identify which transient segments were created from which writers.

Reply:
@tkaitchuck Thoughts?